calabash/Classes/WebViewQuery/LPIsWebView.h

```diff
@@ -1,7 +0,0 @@
-#import <UIKit/UIKit.h>
-
-@interface LPIsWebView : NSObject
-
-+ (BOOL) isWebView:(id) object;
-
-@end
```

Comment:
Next time I think it would be better to change this file name in another PR.

Non blocking.